sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -396,49 +396,14 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
    * @since 1.4.0
    */
   def jdbc(url: String, table: String, connectionProperties: Properties): Unit = {
+    import scala.collection.JavaConverters._
```

Comment:
I opted to only import them here because it is the only place they are required, so there is no need to drag in the import to the whole class.